includes/CreateWiki/SpecialCreateWiki.php

```diff
@@ -5,7 +5,11 @@ public function __construct() {
 	}
 
 	protected function getFormFields() {
-		global $wgCreateWikiUseCategories, $wgCreateWikiCategories, $wgCreateWikiUsePrivateWikis, $wgCreateWikiCDBDirectory;
+		$config = MediaWikiServices::getInstance()->getConfigFactory()->makeConfig( 'createwiki' );
+		$UseCategories = $config->get( 'UseCategories' );
```

Comment:
Convention for variable names is lowercase to upper case, so for example $useCategories. (And for the rest)